messaging/src/main/java/org/axonframework/eventhandling/processors/streaming/pooled/MergeTask.java

```diff
@@ -103,7 +103,7 @@ protected CompletableFuture<Boolean> task() {
                 unitOfWorkFactory
```

Comment:
Again not really a `TokenStore` concern, more an `PooledStreamingEventProcessor` concern. I don't think we need to use the `UnitOfWorkFactory` here, right? Or am I missing something obvious, @MateuszNaKodach?

Reply:
Same as before - everything was the replacement of the TransactionManager. As far as I remember you said - all TokenStore related operation needs to be run in the Transaction (why - to be honest, I don't rememember...).

Reply:
Same reply from me as on the other one. You're right, I simply forgot this decision.